OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -353,9 +375,33 @@ static int MakeALFormat(int channels, int bits)
 
 		public OpenAlSoundSource(byte[] data, int channels, int sampleBits, int sampleRate)
 		{
-			AL10.alGenBuffers(new IntPtr(1), out Buffer);
-			AL10.alBufferData(Buffer, MakeALFormat(channels, sampleBits), data, new IntPtr(data.Length), new IntPtr(sampleRate));
+			AL10.alGenBuffers(new IntPtr(1), out buffer);
+			AL10.alBufferData(buffer, MakeALFormat(channels, sampleBits), data, new IntPtr(data.Length), new IntPtr(sampleRate));
+		}
+
+		#region IDisposable Support
```

Comment:
Nit: We don't have `#region IDisposable Support` in any other code. Please remove it.